src/snippets/glamorous.js

```diff
@@ -127,35 +127,27 @@ const SubmitButton = glamorous.button(
     marginTop: '1rem'
   },
   props => ({
-    backgroundColor: props.disabled ? '#BBB' : undefined
+    backgroundColor: props.disabled ? '#BBB' : null
```

Comment:
This really doesn't make a difference either way, but I prefer `null` to `undefined` 🤷‍♂️